### ISSUE TITLE
Running on Java 12 complains about not supported 1.6

### ISSUE THREAD

**Issue by Zaijo** [chenjianjx/wsdl2html#23]
When calling `generateHtml` of imported 4.0.1 version of this library via maven I get following errors:

```
org.jaxws.wsdl2bytecodes.service.WsdlImportException: 
Compilation Error on line -1 in warning: [options] bootstrap class path not set in conjunction with -source 6
Compilation Error on line -1 in error: Source option 6 is no longer supported. Use 7 or later.
Compilation Error on line -1 in error: Target option 6 is no longer supported. Use 7 or later.
```

What can I do to fix this?

**Comment by chenjianjx** [comment]
1. How did you exactly run it? 
2. If you run it using a command line, please run  `java -version` and `javac -version` to have a check

**Comment by Zaijo** [comment]
I ran it using IDEA run Configuration yielding: 

```
/Library/Java/JavaVirtualMachines/jdk-12.0.2.jdk/Contents/Home/bin/java -ea ....m2/repository/com/github/chenjianjx/wsdl2html/4.0.1/wsdl2html-4.0.1.jar ... com.intellij.rt.execution.junit.JUnitStarter -ideVersion5 -junit4 TestWsdlTypesAndPrintMarkdown,generateWsdlToHtml```

**Comment by chenjianjx** [comment]
You still need to find out your version of javac, because the library will call javac behind the scene

**Comment by Zaijo** [comment]
I'll try to find it. What is the required version of `javac`?

**Comment by Zaijo** [comment]
I didn't make it work as a maven dependency, but at least I have a working solution using https://github.com/chenjianjx/wsdl2html#run-as-a-command-line-tool

**Comment by chenjianjx** [comment]
Fixed in version 4.1.0

**Comment by Zaijo** [comment]
Great thanks :)
